makefile

```diff
@@ -1748,7 +1748,7 @@ $(build)/android.dep: $(luni-javas) $(dalvik-javas) $(libart-javas) \
 		| (cd $(build)/android-src && $(jar) x)
 #	(cd android && $(jar) c *)	| (cd $(build)/android-src && $(jar) x)
 	find $(build)/android-src -name '*.java' > $(build)/android.txt
-	$(javac) -Xmaxerrs 1000 -d $(build)/android @$(build)/android.txt
+	$(javac) -J-Xms256m -J-Xmx256m -Xmaxerrs 1000 -d $(build)/android @$(build)/android.txt
```

Comment:
Looks like it's 256MB here... Strange. I'll reinvestigate that.

    uintx MaxHeapSize                              := 257949696       {product}